src/client/js/Widgets/BranchStatus/BranchStatusWidget.js

```diff
@@ -218,6 +233,26 @@ define([
         this._ddBranchStatus.clear();
         this._outOfSync = true;
         if (this._client.isConnected()) {
+            if (this._config.autoFollowBranchWhenOutOfSync && this._client.getCommitQueue().length === 1) {
```

Comment:
If we allow users to go this direction, probably we can let them go full on, meaning to drop all changes. I would keep the single drop as an option, but offer to drop no matter the amount of changes lost...

Reply:
I don't agree. 
1) The first "out of sync" commit is persisted (when the queue builds up they proceeding ones aren't).
2) With only one commit it should be clear what was dropped (in most cases you'd get direct feedback since you're in that context). Knowing that only one commit can be dropped ensures that the user doesn't have to track back what was dropped.

Reply:
My bad.
Yeah, forgot that here only the branch update fails, but the commit goes through.
I still wonder though if the scenario could even happen and if we should at some point allow scratching that...